src/Documentation/Action/DocumentationAction.php

```diff
@@ -31,32 +31,31 @@ final class DocumentationAction
     private $title;
     private $description;
     private $version;
-    private $formats = [];
+    private $formats;
     private $formatsProvider;
 
     /**
      * @throws InvalidArgumentException
      */
-    public function __construct(ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, string $title = '', string $description = '', string $version = '', /* FormatsProviderInterface */ $formatsProvider = [])
+    public function __construct(ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, string $title = '', string $description = '', string $version = '', $formats = [])
```

Comment:
Can you add a new `$docsFormats` parameter instead please, and revert the changes  related to `$formats`?